landing/index.html

```diff
@@ -4,52 +4,29 @@
     <meta charset="utf-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-    <!--
-      The above 3 meta tags *must* come first in the head; any other head content must come *after* these tags
-    -->
-    <meta
-      name="description"
-      content="End Of Coffee - Track your office inventory"
-    />
+    <meta name="description" content="End Of Coffee" />
     <meta name="author" content="Sandstream Development" />
-
-    <!-- Title -->
-    <title>End Of Coffee - Track Your office inventory</title>
-
-    <!-- Necessary CSS Files -->
-    <!--
-      <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,500,600,700" rel="stylesheet">	 Google Font
-    -->
+    <title>End Of Coffee</title>
```

Comment:
proposal: add more descriptive info?

Reply:
suggest something

Reply:
I think about something similar to description:
**title:**
End of coffee - an app that helps you track all the stuff you care about!" 

This text is better for the title. Description can be and should be a little longer:
**Description:**
End Of Coffe - real-time app for planning and managing your staff. Use it individually or with friends. Create and edit lists, manage cohorts on any device and regardless where you are.

you write it much better than me